---
float64 Width
float64 Height
float64 FocalLength
